README.md

```diff
@@ -85,96 +169,66 @@ And check with `FeatureService.enabled?("some.nested_feature")`.
-bundle exec rake
+# Settings for GOV.UK Notify api & email templates
+govuk_notify:
+  api_key: <API key from Notify>
 ```
 
-## Configuration and deployment
+Example emails can be seen locally by visiting `http://localhost:3000/rails/mailers`
 
-The forms-admin app is containerised (see [Dockerfile](https://github.com/alphagov/forms-admin/blob/main/Dockerfile)) and can be deployed however you would normally deploy a containerised app.
+[GOV.UK Notify]: https://www.notifications.service.gov.uk/
 
-If you are planning to deploy to GOV.UK PaaS without using the container, you can see how this runs in our [Deployment CI action](https://github.com/alphagov/forms-admin/blob/main/.github/workflows/deploy.yml).
+## Configuring Sentry
 
-## Logging
+We use [Sentry] to catch exceptions in production apps and alert us.
 
-- HTTP access logs are managed using [Lograge](https://github.com/roidrage/lograge) and configured within [the application config](./config/application.rb)
-- The output format is JSON using the [JsonLogFormatter](./app/lib/json_log_formatter.rb) to enable simpler searching and visbility especially in Splunk.
-- **DO NOT** use [log_tags](https://guides.rubyonrails.org/configuring.html#config-log-tags) since it breaks the JSON formatting produced by Lograge.
+We currently have a very basic setup for Sentry in this repo for testing, which we will continue to build upon.
 
-## Explain how to test the project
+In order to use this locally you will need to sign up to Sentry and create a new project. Then add the Sentry DSN to your environment as `SETTINGS__SENTRY__DSN`, or add it to a local config file:
 
-```bash
-# Run the Ruby test suite
-bin/rake
-# To run the Javascript test suite, run
-npm run test
 ```
+# config/settings.local.yml
 
-## Explain how to add a user to the database
-
-In order to run this project, your database will need to have a user in it. To add one, run the follwing commands:
-
-```bash
-bin/rails db:seed
+sentry:
+  DSN: <DSN from Sentry>
 ```
 
-## Explain how to use GOV.UK Notify
+If you want to deliberately raise an exception to test, uncomment out the triggers in the [Sentry initializer script](config/initializers/sentry.rb). Whenever you run the app errors will be raised and should also come through on Sentry.
 
-If you want to test the notify function, you will need to get a test API key
-from the [notify service](https://www.notifications.service.gov.uk/) Add it as
-an environment variable under `SETTINGS__GOVUK_NOTIFY__API_KEY=` or creating/edit
-a `config/settings/development.local.yml` and adding the following to it.
+[Sentry]: https://sentry.io
 
-```
-# Settings for GOV.UK Notify api & email templates
-govuk_notify:
-  api_key: KEY_FROM_NOTIFY_SERVICE
-```
+## Deployment
 
-Example emails can be seen locally by visiting `http://localhost:3000/rails/mailers`
+The forms-admin app is containerised (see [Dockerfile](Dockerfile)) and can be deployed however you would normally deploy a containerised app.
```

Comment:
Hmm, I'm not sure if there's a single article that best covers this; it tends to vary from service to service. There is this blog post which is kind of useful, although really it's more about running containers locally:

https://thenewstack.io/containers/how-to-deploy-a-container-with-docker/

But I'd also worry that it might confuse things 🤔